### DISCUSSION TITLE
Specifying nuxt generate cache folder

### DISCUSSION THREAD

**Discussion by francoisauclair911** [nuxt/nuxt#10458] [Questions (Nuxt 2)]
Hi,
It is my understanding that the cache folder used by the `generate` command is located inside the node_modules folder.
Is there a way to specify that folder location ?
I use `.npm` folder in my ci/cd so the `generate` command always restart from zero without using the cache.

Thank you

**Comment by gplusdotgr** [comment]
1. The folder that your generated files end up is configurable with `dir` https://nuxtjs.org/docs/configuration-glossary/configuration-generate/#dir 
2. Have you enabled `cache` ? https://nuxtjs.org/docs/configuration-glossary/configuration-generate/#cache

**Reply by francoisauclair911** [reply]
Hi, 
1- I think the `dir` configuration is not really related to the cache built on `generate` command or am I mistaken
2- It's my understanding that the cache is enabled by default (as the default is an object and not `false`) 

Am I missing something?

**Comment by pencilcheck** [comment]
i'm using netlify and netlify caches node_modules by default, which is a problem because I used lerna and I have shared code that were used as a package dependencies. the common code package is setup as a symlink in node_modules but it seems to be cached on build when deploying to netlify, it would be nice to know if there is a way to have `nuxt generate` to not use cache for specific dependencies.
